src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Methodologies/MethodologyService.cs

```diff
@@ -152,27 +152,62 @@ await _context.Entry(methodology.ScheduledWithRelease)
         public async Task<Either<ActionResult, MethodologySummaryViewModel>> UpdateMethodology(Guid id,
             MethodologyUpdateRequest request)
         {
-            return await _persistenceHelper.CheckEntityExists<Methodology>(id,
-                    HydrateMethodologyForManageMethodologySummaryViewModel)
-                .OnSuccess(methodology => CheckCanUpdateMethodologyStatus(methodology, request.Status))
-                .OnSuccess(_userService.CheckCanUpdateMethodology)
+            return await _persistenceHelper
+                .CheckEntityExists<Methodology>(id, HydrateMethodologyForManageMethodologySummaryViewModel)
+                .OnSuccess(methodology => methodology.Status != request.Status
```

Comment:
There's a problem here which prevents re-approving with  the same status but a different release note, or changing the publishing strategy from scheduled to with a specific release, or changing the specific release to a different release. At the moment it's only checking the status doesn't match the existing status.

Reply:
This is fixed by #2769